### ISSUE TITLE
What is the different between Showcase and Projects Using Nuxt.js?

### ISSUE THREAD

**Issue by ibrahimBeladi** [nuxt-community/awesome-nuxt#475]
What is the different between `Showcase` and `Projects Using Nuxt.js`?
To me, they are the same. If there is a different it would be better to mention it.

Side note, would you mind adding a section (or subsection) for open source projects, which must have their links to their repositories.

**Comment by Geminii** [comment]
Yep, indeed it coud be regroup into the same category as `Projects using Nuxt`.

What do you suggest about adding a new section for open source projects ? 
Can you provide more information ?

**Comment by ibrahimBeladi** [comment]
After merging `Projects using Nuxt` and `Showcase` into something like `Projects using Nuxt (Showcase)`, we can introduce a new section (or a subsection under the merged section) for open source projects exclusively. Where it would only list projects that have their repositories public and open source. 

So, it would consist of 3 parts, name/link and description similar to the current way, with the addition of a link to repository.

**Comment by ibrahimBeladi** [comment]
In order to making the search easier and requires less time. If a user is interested into seeing Nuxt in action, he will navigate to the showcase section. If the user is interested into seeing Nuxt code bases and how they work, he can navigate into the list showing the open source projects and their repositories.

**Comment by Geminii** [comment]
Could be interesting to see it in action 😀
Can you provide a MR ?

**Comment by ibrahimBeladi** [comment]
I can, I will probably merge the two, and introduce a new section for the OSS projects. 
Would you prefer using a table for the new section, or just stick with the bullet points?

**Comment by Geminii** [comment]
Both can be interesting. Maybe you can try to purpose both of them to decide ?
Thanks for it 😀

**Comment by ibrahimBeladi** [comment]
Here you can see how it looks:
- [When nuxt version is used for sectioning](https://github.com/ibrahimBeladi/awesome-nuxt/blob/master/README.md#open-source-projects-using-nuxtjs)
- [When nuxt version is included in the table](https://github.com/ibrahimBeladi/awesome-nuxt/blob/81b27c313fef39791842f1deb913e1b30095eaa7/README.md#open-source-projects-using-nuxtjs)

I did not do the one with bullet points because it is too trivial.
Let me know which one would you prefer so that I make a PR for it.

**Comment by ibrahimBeladi** [comment]
I made a PR for one aspect of the issue (introducing a new section. I will do the first part (merging the two sections) in a later PR.
Feel free to check the PR https://github.com/nuxt-community/awesome-nuxt/pull/486

**Comment by Geminii** [comment]
@ibrahimBeladi thanks a lot for your PR. 
Got some feedbacks and purpose. 

Henceforth, `Nuxt.js` need to be rename in `Nuxt` 😀 (I will update soon all of the readme with these changes).
